code/control/DeployPlanDispatcher.php

```diff
@@ -74,6 +74,7 @@ public function index(\SS_HTTPRequest $request) {
 	public function gitupdate(SS_HTTPRequest $request) {
 		switch($request->httpMethod()) {
 			case 'POST':
+				$this->checkSecurityToken(sprintf('%sSecurityID', get_class($this)));
```

Comment:
ah, good catch, I had it in there, but in my prep for making it a PR it vanished.